dali/pipeline/data/tensor_vector.h

```diff
@@ -229,7 +240,7 @@ class TensorVector {
   std::atomic<int> views_count_;
   std::vector<std::shared_ptr<Tensor<Backend>>> tensors_;
   std::shared_ptr<TensorList<Backend>> tl_;
-  State state_ = State::contiguous;
+  State state_ = State::noncontiguous;
```

Comment:
?

Reply:
By default all the operators use the noncontiguous state (the old way). They ideally would be refactored one by one + we need a small change in executor to set the state as contiguous for Ops that support it.

Reply:
OK